google-cloud-core/src/test/java/com/google/cloud/TimestampTest.java

```diff
@@ -88,6 +88,14 @@ public void ofSqlTimestamp() {
     assertThat(timestamp.toString()).isEqualTo(expectedTimestampString);
   }
 
+  @Test
+  public void testOf_exactSecond() {
+    String expectedTimestampString = "1970-01-01T00:00:12Z";
+    java.sql.Timestamp input = new java.sql.Timestamp(12000);
+    Timestamp timestamp = Timestamp.of(input);
+    assertThat(timestamp.toString()).isEqualTo(expectedTimestampString);
```

Comment:
gratuitous use of assertThat; assertEquals is simpler here

Reply:
Done.